guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/binary-agents/index.md

```diff
@@ -63,10 +63,10 @@ Make sure that each time you transcode a character from binary to decimal, you r
 # Code Explanation:
 
 *   Separate the string into an array of strings separated by whitespace.
-*   Create some variables that will be needed along the way, the names are self explanatory for the most part.
+*   Create some variables that you will use along the way - the names are self explanatory for the most part.
 *   Iterate through each binary string in the new array.
-*   Convert to decimal by using parseInt(_binary_, 2) (with the second parameter we tell in which base our numbers currently are)
-*   At the end, we return out converted message.
+*   Convert to decimal by using `parseInt(_binary_, 2)`. With the second parameter we specify the base of our input numbers.
```

Comment:
Most of your changes look good.  I would suggesting changing the following:
> With the second parameter we specify the base of our input numbers.
to
> Use the second parameter to specify the base of the input number.